sdk/communication/Azure.Communication.NetworkTraversal/src/autorest.md

```diff
@@ -7,29 +7,9 @@ Run `dotnet msbuild /t:GenerateCode` to generate code.
 
 ``` yaml
 title: Network traversal
-tag: package-2021-02-22-preview1
+tag: package-2021-06-21-preview
 model-namespace: false
 require:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/60be518b4fa1a9fb011a0cb69ae7ca3e1cee06b1/specification/communication/data-plane/Microsoft.CommunicationServicesTurn/readme.md
+    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/a59747e42da9d74654106f16c5410620b5941fe0/specification/communication/data-plane/Turn/readme.md
```

Comment:
Should the folder in the spec repo be renamed to NetworkTraversal?

Reply:
Yea I think that will reduce confusion going forward. I'll put out a PR soon renaming it and see what breaking changes I hit